### PR TITLE
[1LP][RFR] Adding Test Coverage for customer BZ

### PR DESCRIPTION
**Pull request by mnadeem92**
Signed-off-by: mnadeem92 <mnadeem@redhat.com>

### CODE REVIEW

File: cfme/tests/test_db_migrate_manual.py [hunk 1]
```diff
@@ -588,3 +588,43 @@ def test_update_webui_replication():
...
+    Polarion:
+        assignee: jhenner
+        casecomponent: Configuration
+        caseimportance: medium
+        initialEstimate: 1h
+        setup: Provision appliance that doesn't have the Embedded Ansible server role enabled
+               upgrade appliance
+               Enable the Embedded Ansible server role
+               Perform a 'yum repolist'.
+    """
```
Comment (ganeshhubale) [inline cfme/tests/test_db_migrate_manual.py]: Add Bugzilla in doc strings too for all tests.

File: cfme/tests/cli/test_appliance_console.py [hunk 2]
```diff
@@ -1321,3 +1321,29 @@ def test_appliance_console_negative():
             1. Check info screen is blank with Network information.
     """
     pass
+
+
+@pytest.mark.manual
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1753687])
+def test_failed_command_status():
```
Comment (digitronik) [inline cfme/tests/cli/test_appliance_console.py]: Q: isn't it more generic test name? I would try to give specific name.

File: cfme/tests/test_db_migrate_manual.py [hunk 3]
```diff
@@ -588,3 +588,59 @@ def test_update_webui_replication():
...
+        1678337
+    Polarion:
+        assignee: jhenner
+        casecomponent: Configuration
+        caseimportance: medium
+        initialEstimate: 1h
+        testSteps:
+            1. Provision appliance
+            2. Subscribe to target version channels and update the packages
+            3. Run rake db:migrate command and wait for it complete
```
Comment (digitronik) [inline cfme/tests/test_db_migrate_manual.py]: Q: I'm suspect db migration is part of update or upgrade? If upgrade i would try to give receptive name.

File: cfme/tests/test_db_migrate_manual.py [hunk 4]
```diff
@@ -588,3 +588,59 @@ def test_update_webui_replication():
...
+            4. Embedded ansible role should start successfully
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.update
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1678614])
+def test_embedded_ansible_repos():
```
Comment (digitronik) [inline cfme/tests/test_db_migrate_manual.py]: same ^^ i would try to give specific test name not general.

**Pull request merged**